azure/template.json

```diff
@@ -30,12 +30,30 @@
         },
         "containerArray": {
             "type": "array"
+        },
+        "workerAspSize": {
+            "type": "string",
+            "defaultValue": "1"
+        },
+        "workerAspInstances": {
+            "type": "int",
+            "defaultValue": 1
+        },
+        "loggingRedisConnectionString": {
+            "type": "securestring"
+        },
+        "configurationStorageConnectionString": {
+            "type": "securestring"
         }
     },
     "variables": {
         "deploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/das-platform-building-blocks/master/templates/",
         "resourceNamePrefix": "[toLower(concat('das-', parameters('resourceEnvironmentName'),'-', parameters('serviceName')))]",
         "storageAccountName": "[toLower(concat('das', parameters('resourceEnvironmentName'), parameters('serviceName'), 'str'))]",
+        "workerAppServicePlanName": "[concat(variables('resourceNamePrefix'), 'wkr-asp')]",
+        "workerFunctionAppStorageAccountName": "[concat('das', parameters('resourceEnvironmentName'), parameters('serviceName'),'wkrstr')]",
```

Comment:
Could we not share a Storage account. Especially as we are running close to maximum Storage account limits constantly.

Reply:
Agreed, I've updated the ARM template to use the single storage account for the ETL containers and the Function App logs.